lib/src/main/java/uk/gov/di/data/lep/library/dto/DeathNotification/IsoDate.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.di.data.lep.library.dto.DeathNotification;
+
+import java.time.LocalDate;
+
+public record IsoDate(
```

Comment:
Appreciate IsoDate is what it's called in the json schema, but it feels like a really bad name here, it's really a "DateWIthDescription"